### DISCUSSION TITLE
Question: I don't think this is supposed to happen.

### DISCUSSION THREAD

**Discussion by IndigoVoidigo** [FunkinCrew/Funkin#1542] [Q&A] [answered]
## What is your question?

Why is it that some symbols for NOTE_Assets.fla are named differently than in the original NOTE_Assets.xml file? Also, the sprite sheet isn't the same as the original sprite sheet as well.

**Comment by Lasercar** [comment, marked as answer]
Artists

**Reply by IndigoVoidigo** [reply]
bro.
it has been like 4 years.
